### DISCUSSION TITLE
不同的浏览器播放行为不一致

### DISCUSSION THREAD

**Discussion by zengyangyang** [Tencent/libpag#2281] [General]
使用几个版本尝试都有问题，例如v4.3.51 v4.2.84
PAG文件存在BMP混合导出，总时长5s，前3s是完全透明没有任何内容
现象是无限重播的时候，Chrome浏览器透明3s，有画面2s正常循环，Safari浏览器播放完第一遍之后画面静止3s然后播放2s动画，停留在最后一帧3s，2s动画循环。不会恢复到完全无画面的状态。打印currentframe看起来是正常的，PAG官网预览界面可复现

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.3.51 / 4.2.84。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
